apps/site/pages/en/learn/modules/package-configuration.mdx

```diff
@@ -0,0 +1,597 @@
+---
+title: Package Configuration
+layout: learn
+authors: JakobJingleheimer
+---
+
+# Package configuration
```

Comment:
I'd start the guide with the topic this article is talking about:
How to publish a package, which might be obvious for some but is not clearly stated